### ISSUE TITLE
Can't move my mouse up on multiplayer / zombies

### ISSUE THREAD

**Issue by BianchettoS1X** [XLabsProject/s1x-client#111]
Hi! 
I just wanted to report a problem that just me and another person have!
Basically when i try to play singleplayer on S1X I don't have any problems, but when i try to play mulitplayer or zombies i can move my mouse (in-game) just right, left, down but not up.
I tried literally everything, after troubleshooting for hours i just stopped, i think it is an S1X problem cause when i play singleplayer it just works...

Thanks for the help and the opportunity to contact you,
Have a nice day!

**Comment by Brentdevent** [comment]
![image](https://user-images.githubusercontent.com/70229620/116465311-58398b80-a86d-11eb-9139-5863197e1d00.png)

https://user-images.githubusercontent.com/70229620/116465179-2e806480-a86d-11eb-8874-048439b8f14b.mp4

Recorded by Starnalz#9432. I assume it gives a better insight in a possible fix.

**Comment by BianchettoS1X** [comment]
It could be like, i don't know, it does that while i'm fullscreen too?

**Comment by Starnalz** [comment]
I would like to add another thing, when i launch the game i have to alt tab into it and when i get into the actual game like private match or public, i cant move my mouse at all, i have to put my game in fullscreen then windowed then borderless, and then this starts to happen

**Comment by ZachG2020** [comment]
> Hi!
> I just wanted to report a problem that just me and another person have!
> Basically when i try to play singleplayer on S1X I don't have any problems, but when i try to play mulitplayer or zombies i can move my mouse (in-game) just right, left, down but not up.
> I tried literally everything, after troubleshooting for hours i just stopped, i think it is an S1X problem cause when i play singleplayer it just works...
> 
> Thanks for the help and the opportunity to contact you,
> Have a nice day!

I had the same issue, I ended up fixing the problem on my computer by turning off Nvidia Broadcast and any other Nvidia application from the system tray.

1) Go to the system tray(If you don't know what that is, look at the bottom right of the screen next to the date and time you will see a "^" symbol)
2) Right Click on any Nvidia Application, and there should be an option to Exit, Quit...etc, Do that for every Nvidia Application present.
3) Restart the game (Game only, no need for system restart)
4) Enjoy :)

**Comment by ZachG2020** [comment]
> ![image](https://user-images.githubusercontent.com/70229620/116465311-58398b80-a86d-11eb-9139-5863197e1d00.png)
> 
>  why_is_this_happening.mp4 
> Recorded by Starnalz#9432. I assume it gives a better insight in a possible fix.
> 
> Also the use of a controller doesn't seem to solve the problem.

I solved this problem by tabbing(if you are using mouse/keyboard), to see the scoreboard and then releasing which solved the issue. If it still persists do ALT + ENTER, to window the application and then do it again to go back to Fullscreen!

**Comment by momo5502** [comment]
> ![image](https://user-images.githubusercontent.com/70229620/116465311-58398b80-a86d-11eb-9139-5863197e1d00.png)
> 
>  why_is_this_happening.mp4 
> Recorded by Starnalz#9432. I assume it gives a better insight in a possible fix.
> 
> Also the use of a controller doesn't seem to solve the problem.

I see some FPS counter that is not supposed to be in there. Probably some garbage 3rd party software hooking the game badly causing this.

**Comment by Brentdevent** [comment]
It seems indeed being caused by 3rd-party software.  Both of the people having this issue were contacted. They managed to fix it. (@Starnalz through fresh install most likely removing this 3rd-party FPS counter). @BianchettoS1X followed the tutorial from @OfficialGodAwful-aka-Krater. (Thanks for sharing the solution!)

Therefore it seems this issue can be closed.

**Comment by Inhumanthree1** [comment]
This issue occurs for me too, but I don't have any possible 3rd party software that I could think of running. No discord, no FRAPS, no browser, no NVidia GFE/Broadcast, I even turned off the Xbox Game Bar.
![image](https://user-images.githubusercontent.com/6027695/121104057-3833ab00-c7cf-11eb-9e29-b48e6ef9bdb8.png)

![image](https://user-images.githubusercontent.com/6027695/121104071-3e298c00-c7cf-11eb-8657-daf3d3d87ea7.png)

![image](https://user-images.githubusercontent.com/6027695/121104092-4b467b00-c7cf-11eb-9bc4-cdab2b56201b.png)

the only things open other than this are windows processes (Discord was open in this screenshot as I was using it to communicate but I closed it before opening the game. The issue still persists.)

Additionally, when I had first started playing there was no mouse input whatsoever. Only after a few restarts did it allow me to move down and right.
